### PR TITLE
Updated material-branch to changes from master https://github.com/google/WebFundamentals/pull/1875

### PR DESCRIPTION
**Pull request by johyphenel**
Merging changes from https://github.com/google/WebFundamentals/pull/1875 to material branch.

### CODE REVIEW

**Comment by johyphenel** [comment]
CLA covered by Google corp - just signed individual one as well.

File: src/content/en/fundamentals/security/avoid-mixed-content/_code/active-mixed-content.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- // [TEMPLATE header_full] -->
```
Comment (johyphenel) [inline src/content/en/fundamentals/security/avoid-mixed-content/_code/active-mixed-content.html]: @petele - should I be removing this markup from master?

File: src/content/en/fundamentals/security/avoid-mixed-content/_code/active-mixed-content.html [hunk 2]
```diff
@@ -38,42 +39,46 @@
...
-    <p>
-      Several examples of active mixed content. When viewed over HTTPS most browsers block this content and display errors in the JavaScript console.
-    </p>
-    <div class="insecure-style-holder">
-      <span style="ba">Insecure style loaded</span>
-    </div>
-    <div class="insecure-background">
-      Loading insecure background here...
-    </div>
+    <div role="main">
```
Comment (johyphenel) [inline src/content/en/fundamentals/security/avoid-mixed-content/_code/active-mixed-content.html]: @petele ditto?

**Comment by petele** [comment]
I manually merged this as #1880 because of other merging issues.

**Pull request closed without merging**